src/main/kotlin/simulator/io/Input.kt

```diff
@@ -1,12 +1,19 @@
 package simulator.io
 
+import java.util.*
+
 class Input {
     fun getMoney(): Int {
         println("구입금액을 입력해 주세요.")
         return readln().toInt()
     }
 
-    fun getWinningNumbers(): Set<Int> {
+    fun getBonusNumber(): Int {
+        println("보너스 볼을 입력해 주세요.")
+        return readln().toInt()
+    }
+
+    fun getWinningNumbers(): SortedSet<Int> {
```

Comment:
```suggestion
    fun getWinningNumbers(): Set<Int> {
```
코틀린에서의 Set은 내부적으로 순서를 보장해요.
```java
public fun <T> Iterable<T>.toSet(): Set<T> {
    if (this is Collection) {
        return when (size) {
            0 -> emptySet()
            1 -> setOf(if (this is List) this[0] else iterator().next())
            else -> toCollection(LinkedHashSet<T>(mapCapacity(size)))
        }
    }
    return toCollection(LinkedHashSet<T>()).optimizeReadOnlySet()
}
```
set으로 변환시키는 `toSet` 함수를 보면, `LinkedHashSet`으로 변환해주는 것을 볼 수 있어요.
LinkedHashSet의 특징에 대해서 학습해보면 어떨까요?